public/js/user.js

```diff
@@ -450,5 +539,21 @@ function contentEditwindow(site) {
   html += "</select></div>";
   html += "<input type='hidden' id='site' value='"+site.id+"'/>";
   html += "<div class=''><button type='submit' class='btn btn-primary'>Save</button></div></div></form></div></div>";
+
+  if((site.created_by === currentUser || isAdmin) && isValidMarkerId(site.id)) {
+    html += "<div id='adv-button'><span class='glyphicon glyphicon glyphicon-cog'></span> Advanced Options &nbsp; <span class='glyphicon glyphicon-chevron-down' id='adv-icon'></span></div>"
+    html += "<div id='adv-options'>"
+    
+    html += "<div class='form-inline'><br>Co-owner"
+    html += "<input class='form-control' title='Co-owner' name='coowner' id ='coowner' style='width:80px; margin:5px;'>"
+    html += "<button id='addcoowner' class='btn btn-primary'>Add</button>";
```

Comment:
Ideally, the <kbd>Add</kbd> button would be disabled as long as the co-owner field is empty so users couldn't easily submit an empty co-owner request.